### PR TITLE
Remunerations payments for Jan 2023

### PR DESCRIPTION
**Pull request by hannespfeiffer**
This script sends the last remaining batch of remunerations of ~ 1.69m SDL to affected parties according to [SIP-21](https://snapshot.org/#/saddlefinance.eth/proposal/0xf27b07ef7025aa23edb2a84d56825511ea7adff8d6164cd09a23c63c42cdc01a). 

Tx logs:

```
Transaction was Mined 
---------------------
Tx Hash: 0xdaa6d2e71533f873fe7547d30fd20be8efa99b38edf98610a0a9fc0a5f163567
From: 0x0AF91FA049A7e1894F480bFE5bBa20142C6c29a9
To: 0x3F8E527aF4e0c6e763e8f368AC679c44C45626aE
Value: 0
Function: Gnosis Safe.execTransaction
Block: 16571036
Gas Used: 435563 / 446516 (97.5%)

Events In This Transaction
--------------------------
├── Saddle DAO (0xf1Dc500FdE233A4055e25e5BbF516372BC4F6871)
│   ├── Transfer
│   │   ├── from: 0x3F8E527aF4e0c6e763e8f368AC679c44C45626aE
│   │   ├── to: 0xa0f75491720835b36edC92D06DDc468D201e9b73
│   │   └── value: 388195348036185000000000
│   ├── Transfer
│   │   ├── from: 0x3F8E527aF4e0c6e763e8f368AC679c44C45626aE
│   │   ├── to: 0x38eE5F5A39c01cB43473992C12936ba1219711ab
│   │   └── value: 377228252453672500000000
│   ├── Transfer
│   │   ├── from: 0x3F8E527aF4e0c6e763e8f368AC679c44C45626aE
│   │   ├── to: 0x38f9aBD4BD8947dA035abf51fC21244108a81CA6
│   │   └── value: 283110637209162530000000
│   ├── Transfer
│   │   ├── from: 0x3F8E527aF4e0c6e763e8f368AC679c44C45626aE
│   │   ├── to: 0x0EA4A285E1353F490eC7f473AB3174Cac71cF79a
│   │   └── value: 283054983924229980000000
│   ├── Transfer
│   │   ├── from: 0x3F8E527aF4e0c6e763e8f368AC679c44C45626aE
│   │   ├── to: 0x6eF3f0fF397C3de367C6cA2Cc3e6B6955E7993A5
│   │   └── value: 45814168072575000000000
│   ├── Transfer
│   │   ├── from: 0x3F8E527aF4e0c6e763e8f368AC679c44C45626aE
│   │   ├── to: 0x6a73204dB71F8e054bf9A0680b02Ae96f700b595
│   │   └── value: 45316406746405000000000
│   ├── Transfer
│   │   ├── from: 0x3F8E527aF4e0c6e763e8f368AC679c44C45626aE
│   │   ├── to: 0x5eeA118E75f247014c6D0E990f02a0c254EDC852
│   │   └── value: 37829766448069000000000
│   ├── Transfer
│   │   ├── from: 0x3F8E527aF4e0c6e763e8f368AC679c44C45626aE
│   │   ├── to: 0x2B9D8f558e02753ed7D4D97DEdfAd662C02af3CB
│   │   └── value: 37749640861081498000000
│   ├── Transfer
│   │   ├── from: 0x3F8E527aF4e0c6e763e8f368AC679c44C45626aE
│   │   ├── to: 0x0BB31C6278d58CfF41B7e8Ed3B20f76424fD69Ad
│   │   └── value: 37734050383692500000000
│   ├── Transfer
│   │   ├── from: 0x3F8E527aF4e0c6e763e8f368AC679c44C45626aE
│   │   ├── to: 0xc7D2a419f8f2eAa06B3811cF7faAa7440DCf96b7
│   │   └── value: 37722144931399500000000
│   ├── Transfer
│   │   ├── from: 0x3F8E527aF4e0c6e763e8f368AC679c44C45626aE
│   │   ├── to: 0xAe1E5C84B73136a49eFb70a15149c350015391eD
│   │   └── value: 28313813312198503000000
│   ├── Transfer
│   │   ├── from: 0x3F8E527aF4e0c6e763e8f368AC679c44C45626aE
│   │   ├── to: 0xE2d079278c0f4959199136F8db11c645d69A292b
│   │   └── value: 28308710977107500000000
│   ├── Transfer
│   │   ├── from: 0x3F8E527aF4e0c6e763e8f368AC679c44C45626aE
│   │   ├── to: 0xffd4DAe0d7D8ddb6F408dca0a47763Ae3A57F4Ce
│   │   └── value: 15570159540199500000000
│   ├── Transfer
│   │   ├── from: 0x3F8E527aF4e0c6e763e8f368AC679c44C45626aE
│   │   ├── to: 0x9c7d894a29F97636B66DEA646A1AE2F9c83a2830
│   │   └── value: 13612657369877500000000
│   ├── Transfer
│   │   ├── from: 0x3F8E527aF4e0c6e763e8f368AC679c44C45626aE
│   │   ├── to: 0x77956a9D1b9901BbE4CFC2ED37DB17500Fbe7B0C
│   │   └── value: 9422032340562500000000
│   ├── Transfer
│   │   ├── from: 0x3F8E527aF4e0c6e763e8f368AC679c44C45626aE
│   │   ├── to: 0x02eF8147E2D0997cCA48d99f01bAD846D16558fa
│   │   └── value: 7542012746875000000000
│   ├── Transfer
│   │   ├── from: 0x3F8E527aF4e0c6e763e8f368AC679c44C45626aE
│   │   ├── to: 0xC7193C01f43257F3FE4C8f86784FcAA578A9e2b1
│   │   └── value: 6034739792375001000000
│   ├── Transfer
│   │   ├── from: 0x3F8E527aF4e0c6e763e8f368AC679c44C45626aE
│   │   ├── to: 0xd716ad4f10eade5aF4Ed10006678f49Bd2E6624f
│   │   └── value: 1886350382875000000000
│   ├── Transfer
│   │   ├── from: 0x3F8E527aF4e0c6e763e8f368AC679c44C45626aE
│   │   ├── to: 0x99eb33756a2eAa32f5964A747722c4b59e6aF351
│   │   └── value: 803715183375000000000
│   ├── Transfer
│   │   ├── from: 0x3F8E527aF4e0c6e763e8f368AC679c44C45626aE
│   │   ├── to: 0x2a1EaF114f2c23d30EF571d6E84fCBeb2f6F4e62
│   │   └── value: 377475764000000050000
│   ├── Transfer
│   │   ├── from: 0x3F8E527aF4e0c6e763e8f368AC679c44C45626aE
│   │   ├── to: 0x46866D274E6D9015c5FDc098CE270803e11e3eF4
│   │   └── value: 207497024750000000000
│   ├── Transfer
│   │   ├── from: 0x3F8E527aF4e0c6e763e8f368AC679c44C45626aE
│   │   ├── to: 0x2d5D79753bB8e02ca1ce8EcE0aa55288F8c8d840
│   │   └── value: 103748512375000000000
│   ├── Transfer
│   │   ├── from: 0x3F8E527aF4e0c6e763e8f368AC679c44C45626aE
│   │   ├── to: 0xb5E46aFa9596cB6F44d2Fcb6Dbd2096dbe35396C
│   │   └── value: 18989557625000000000
│   └── Transfer
│       ├── from: 0x3F8E527aF4e0c6e763e8f368AC679c44C45626aE
│       ├── to: 0x74F8744020f5900a2C3A3289C6c2182F56A6901f
│       └── value: 5198384374999999000
│   
└── Gnosis Safe (0x3F8E527aF4e0c6e763e8f368AC679c44C45626aE)
    └── ExecutionSuccess
        ├── txHash: 0x7ec0b50977ef5c153770891192fd56d86850df98e4a38a07ecc0403ef96db27e
        └── payment: 0
```

### CODE REVIEW

**Review by penandlim — APPROVED** [review]

**Review by penandlim — CHANGES_REQUESTED** [review]
Need to merge from origin/master so saddle-contract folder is not being rolledback to previous versions

**Comment by penandlim** [comment]
Could we also add claiming from vesting contract and sending 2_000_000 to Minter contract to this TX as well?

File: scripts/2023_02_06_remunerations_January_plus_2m_minter_refill.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+    sdl_vesting_contract_proxy = multisig.get_contract(
+        SDL_DAO_COMMUNITY_VESTING_PROXY_ADDRESS[CHAIN_IDS[TARGET_NETWORK]]
+    )
+
+    # Claim SDL from vesting contract
+    sdl_vesting_contract_proxy.release()
+
+    # Transfer 2m SDL to minter
+    sdl_contract.transfer(SDL_MINTER_ADDRESS[CHAIN_IDS["MAINNET"]], 2_000_000)
```
Comment (penandlim) [inline scripts/2023_02_06_remunerations_January_plus_2m_minter_refill.py:82]: Missing 1e18 multiplier for the token amount

**Review by penandlim — CHANGES_REQUESTED** [review]
please do `git fetch && git merge origin/master` to prevent saddle-contract overwrite!

or you can also exclude saddle-contract in this PR.

**Comment by hannespfeiffer** [comment]
Tx logs including claiming SDL from vesting contract and sending 2m SDL to minter contract:


```
Transaction was Mined 
---------------------
Tx Hash: 0x20ae30683dfa5986f6cb7b7a5b2e1ba809a577309a01fe76e113551a9bc1bde8
From: 0x0AF91FA049A7e1894F480bFE5bBa20142C6c29a9
To: 0x3F8E527aF4e0c6e763e8f368AC679c44C45626aE
Value: 0
Function: Gnosis Safe.execTransaction
Block: 16571364
Gas Used: 482455 / 495317 (97.4%)

Events In This Transaction
--------------------------
├── Saddle DAO (0xf1Dc500FdE233A4055e25e5BbF516372BC4F6871)
│   ├── Transfer
│   │   ├── from: 0x3F8E527aF4e0c6e763e8f368AC679c44C45626aE
│   │   ├── to: 0xa0f75491720835b36edC92D06DDc468D201e9b73
│   │   └── value: 388195348036185000000000
│   ├── Transfer
│   │   ├── from: 0x3F8E527aF4e0c6e763e8f368AC679c44C45626aE
│   │   ├── to: 0x38eE5F5A39c01cB43473992C12936ba1219711ab
│   │   └── value: 377228252453672500000000
│   ├── Transfer
│   │   ├── from: 0x3F8E527aF4e0c6e763e8f368AC679c44C45626aE
│   │   ├── to: 0x38f9aBD4BD8947dA035abf51fC21244108a81CA6
│   │   └── value: 283110637209162530000000
│   ├── Transfer
│   │   ├── from: 0x3F8E527aF4e0c6e763e8f368AC679c44C45626aE
│   │   ├── to: 0x0EA4A285E1353F490eC7f473AB3174Cac71cF79a
│   │   └── value: 283054983924229980000000
│   ├── Transfer
│   │   ├── from: 0x3F8E527aF4e0c6e763e8f368AC679c44C45626aE
│   │   ├── to: 0x6eF3f0fF397C3de367C6cA2Cc3e6B6955E7993A5
│   │   └── value: 45814168072575000000000
│   ├── Transfer
│   │   ├── from: 0x3F8E527aF4e0c6e763e8f368AC679c44C45626aE
│   │   ├── to: 0x6a73204dB71F8e054bf9A0680b02Ae96f700b595
│   │   └── value: 45316406746405000000000
│   ├── Transfer
│   │   ├── from: 0x3F8E527aF4e0c6e763e8f368AC679c44C45626aE
│   │   ├── to: 0x5eeA118E75f247014c6D0E990f02a0c254EDC852
│   │   └── value: 37829766448069000000000
│   ├── Transfer
│   │   ├── from: 0x3F8E527aF4e0c6e763e8f368AC679c44C45626aE
│   │   ├── to: 0x2B9D8f558e02753ed7D4D97DEdfAd662C02af3CB
│   │   └── value: 37749640861081498000000
│   ├── Transfer
│   │   ├── from: 0x3F8E527aF4e0c6e763e8f368AC679c44C45626aE
│   │   ├── to: 0x0BB31C6278d58CfF41B7e8Ed3B20f76424fD69Ad
│   │   └── value: 37734050383692500000000
│   ├── Transfer
│   │   ├── from: 0x3F8E527aF4e0c6e763e8f368AC679c44C45626aE
│   │   ├── to: 0xc7D2a419f8f2eAa06B3811cF7faAa7440DCf96b7
│   │   └── value: 37722144931399500000000
│   ├── Transfer
│   │   ├── from: 0x3F8E527aF4e0c6e763e8f368AC679c44C45626aE
│   │   ├── to: 0xAe1E5C84B73136a49eFb70a15149c350015391eD
│   │   └── value: 28313813312198503000000
│   ├── Transfer
│   │   ├── from: 0x3F8E527aF4e0c6e763e8f368AC679c44C45626aE
│   │   ├── to: 0xE2d079278c0f4959199136F8db11c645d69A292b
│   │   └── value: 28308710977107500000000
│   ├── Transfer
│   │   ├── from: 0x3F8E527aF4e0c6e763e8f368AC679c44C45626aE
│   │   ├── to: 0xffd4DAe0d7D8ddb6F408dca0a47763Ae3A57F4Ce
│   │   └── value: 15570159540199500000000
│   ├── Transfer
│   │   ├── from: 0x3F8E527aF4e0c6e763e8f368AC679c44C45626aE
│   │   ├── to: 0x9c7d894a29F97636B66DEA646A1AE2F9c83a2830
│   │   └── value: 13612657369877500000000
│   ├── Transfer
│   │   ├── from: 0x3F8E527aF4e0c6e763e8f368AC679c44C45626aE
│   │   ├── to: 0x77956a9D1b9901BbE4CFC2ED37DB17500Fbe7B0C
│   │   └── value: 9422032340562500000000
│   ├── Transfer
│   │   ├── from: 0x3F8E527aF4e0c6e763e8f368AC679c44C45626aE
│   │   ├── to: 0x02eF8147E2D0997cCA48d99f01bAD846D16558fa
│   │   └── value: 7542012746875000000000
│   ├── Transfer
│   │   ├── from: 0x3F8E527aF4e0c6e763e8f368AC679c44C45626aE
│   │   ├── to: 0xC7193C01f43257F3FE4C8f86784FcAA578A9e2b1
│   │   └── value: 6034739792375001000000
│   ├── Transfer
│   │   ├── from: 0x3F8E527aF4e0c6e763e8f368AC679c44C45626aE
│   │   ├── to: 0xd716ad4f10eade5aF4Ed10006678f49Bd2E6624f
│   │   └── value: 1886350382875000000000
│   ├── Transfer
│   │   ├── from: 0x3F8E527aF4e0c6e763e8f368AC679c44C45626aE
│   │   ├── to: 0x99eb33756a2eAa32f5964A747722c4b59e6aF351
│   │   └── value: 803715183375000000000
│   ├── Transfer
│   │   ├── from: 0x3F8E527aF4e0c6e763e8f368AC679c44C45626aE
│   │   ├── to: 0x2a1EaF114f2c23d30EF571d6E84fCBeb2f6F4e62
│   │   └── value: 377475764000000050000
│   ├── Transfer
│   │   ├── from: 0x3F8E527aF4e0c6e763e8f368AC679c44C45626aE
│   │   ├── to: 0x46866D274E6D9015c5FDc098CE270803e11e3eF4
│   │   └── value: 207497024750000000000
│   ├── Transfer
│   │   ├── from: 0x3F8E527aF4e0c6e763e8f368AC679c44C45626aE
│   │   ├── to: 0x2d5D79753bB8e02ca1ce8EcE0aa55288F8c8d840
│   │   └── value: 103748512375000000000
│   ├── Transfer
│   │   ├── from: 0x3F8E527aF4e0c6e763e8f368AC679c44C45626aE
│   │   ├── to: 0xb5E46aFa9596cB6F44d2Fcb6Dbd2096dbe35396C
│   │   └── value: 18989557625000000000
│   └── Transfer
│       ├── from: 0x3F8E527aF4e0c6e763e8f368AC679c44C45626aE
│       ├── to: 0x74F8744020f5900a2C3A3289C6c2182F56A6901f
│       └── value: 5198384374999999000
│   
├── Vesting (0x5DFbCeea7A5F6556356C7A66d2A43332755D68A5)
│   └── Released
│       └── amount: 1064228183663115169964485
│   
├── Saddle DAO (0xf1Dc500FdE233A4055e25e5BbF516372BC4F6871)
│   ├── Transfer
│   │   ├── from: 0x5DFbCeea7A5F6556356C7A66d2A43332755D68A5
│   │   ├── to: 0x3F8E527aF4e0c6e763e8f368AC679c44C45626aE
│   │   └── value: 1064228183663115169964485
│   └── Transfer
│       ├── from: 0x3F8E527aF4e0c6e763e8f368AC679c44C45626aE
│       ├── to: 0x358fE82370a1B9aDaE2E3ad69D6cF9e503c96018
│       └── value: 2000000
│   
└── Gnosis Safe (0x3F8E527aF4e0c6e763e8f368AC679c44C45626aE)
    └── ExecutionSuccess
        ├── txHash: 0x97c5192713267ae8eb8e8cc7746c78149a495f4c38bc84917868f96164b6a55b
        └── payment: 0

**Pull request closed without merging**